ui/widgets/layout/src/components/layout-widget.tsx

```diff
@@ -23,9 +23,10 @@ import {
   FocusedPluginSlot,
   CookieWidgetSlot,
 } from './styled-slots';
-import { usePlaformHealthCheck } from '@akashaorg/ui-awf-hooks';
+import { usePlaformHealthCheck, useCloseNotification } from '@akashaorg/ui-awf-hooks';
 
 const { Box, BasicCardBox, Icon, styled, Text, Extension } = DS;
+const CLOSE_THE_MERGE_NOTIFICATION_FLAG = 'close-the-merge-notification';
```

Comment:
there should be a mapping of the registered flags when these are passed to the hook, otherwise there could be multiple components registering the same flag name

Reply:
Thanks. I have implemented it in a different way :)